__tests__/api.test.js

```diff
@@ -0,0 +1,33 @@
+const app = require('../app.js');
+const request = require('supertest');
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const format = require("pg-format");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+describe('app', () => {
+  describe("api/topics", () => {
+    test("Status 200: with a message on the body", () => {
+      return request(app).get("/api/topics").expect(200);
+    });
+    test("200: responds with an array of topic object", () => {
+      return request(app).get("/api/topics").expect(200).then((response) => {
+       const { topics } = response.body;
+       expect(topics).toBeInstanceOf(Array);
```

Comment:
We know that the topics are an array because you can successfully use a forEach, so this has to be an array. Testing for the data type is unnecessary. 

As you're using a forEach, there should also be an assertion for the length of the array, how many are we expecting back from the database?